ox-jira.el

```diff
@@ -119,7 +119,7 @@
     (keyword . (lambda (&rest args) ""))
     (latex-environment . (lambda (&rest args) (ox-jira--not-implemented 'latex-environment)))
     (latex-fragment . (lambda (&rest args) (ox-jira--not-implemented 'latex-fragment)))
-    (line-break . (lambda (&rest args) (ox-jira--not-implemented 'line-break)))
+    (line-break . (lambda (&rest args) "\n\n"))
```

Comment:
According to https://jira.atlassian.com/secure/WikiRendererHelpAction.jspa?section=breaks `\\` should create a linebreak. I think that might be more accurate than `\n\n`, which appears to be a paragraph break?